TEAMWORK.md

```diff
@@ -0,0 +1,28 @@
+# Teamwork Contract
+
+## Division of Labour
+- **@jstet** will be responsible for:
```

Comment:
If u guys are ok with it, I can also fix one of the errors in the .Rmd (someone else could add it to the repo)